common/djangoapps/student/check.py

```diff
@@ -1,4 +0,0 @@
-from django.utils.http import int_to_base36
```

Comment:
check.py is not part of master branch. You should remove it as it seems you added it for your testing.

Reply:
I already deleted it